test/functional/api/v2/users_controller_test.rb

```diff
@@ -1,6 +1,7 @@
 require 'test_helper'
 
 class Api::V2::UsersControllerTest < ActionController::TestCase
+  AuthSourceInternal.create(:name => "Internal") unless AuthSourceInternal.first.present?
```

Comment:
There's a fixture already creating an internal AuthSource,`auth_sources(:internal)`, with the exact same name even.